daffodil-test/src/test/scala/org/apache/daffodil/section05/simple_types/TestSimpleTypes2.scala

```diff
@@ -99,4 +99,6 @@ class TestSimpleTypes2 {
 
   @Test def test_hexBinary_Delimited_04(): Unit = { runner.runOneTest("hexBinary_Delimited_04") }
 
+  @Test def test_restrictionBaseEmbeddedLeadlingSpace(): Unit = {runner.runOneTest("restrictionBaseEmbeddedLeadingSpace")}
+  @Test def test_restrictionBaseEmbeddedTrailingSpace(): Unit = {runner.runOneTest("restrictionBaseEmbeddedTrailingSpace")}
```

Comment:
Add spaces inside the curly braces.

Reply:
Great comments, Steve. I believe I have addressed all of them.